src/test/ui/rfc1598-generic-associated-types/generic_associated_types_equals.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(generic_associated_types)]
+
+trait Foo {
+    type Bar<T=usize>;
```

Comment:
This form of default we will *definitely* not support.

Reply:
> This form of default we will definitely not support.

This is the kind of information I was looking for since that will allow me to figure out which test cases I need to add to make sure we explicitly reject it. :)